rest_golang/mongo/connectionhelper.go

```diff
@@ -0,0 +1,46 @@
+package mongo
+
+import (
+	"context"
+	"sync"
+
+	"go.mongodb.org/mongo-driver/mongo"
+	"go.mongodb.org/mongo-driver/mongo/options"
+)
+
+/* Used to create a singleton object of MongoDB client.
+Initialized and exposed through  GetMongoClient().*/
+var clientInstance *mongo.Client
+
+//Used during creation of singleton client object in GetMongoClient().
+var clientInstanceError error
+
+//Used to execute client creation procedure only once.
+var mongoOnce sync.Once
+
+//I have used below constants just to hold required database config's.
+const (
+	// CONNECTIONSTRING = "mongodb://localhost:27017"
+	CONNECTIONSTRING = "mongodb://root:rootpassword@localhost:27017/?authSource=admin"
```

Comment:
Can this be moved to the .env file in the root directory of the project if possible? Let me know.

Reply:
Moved to .env